setup_cosim_build_env.sh

```diff
@@ -44,8 +44,9 @@ else
 fi
 
 # Build COSIM runtime library and simulation executable
+export BSG_MACHINE=4x4_fast_n_fake
```

Comment:
this will override the settings in `bsg_replicant/machine.mk` right?

Reply:
Yes, just wanted to be able to set the machine in pytorch, independent of what's set in bsg_replicant.